lua/README.md

```diff
@@ -0,0 +1,172 @@
+# Lua 5.4 on Unikraft
+
+Build and run a Lua 5.4 program on Unikraft.  
+Follow the instructions below to set up, configure, build and run Lua.  
+Make sure you installed the [requirements](../README.md#requirements).
+
+## Quick Setup (aka TLDR)
+
+For a quick setup, run the commands below.  
+Note that you still need to install the [requirements](../README.md#requirements).  
+Before everything, make sure you run the top-level `setup.sh` script:
+
+```console
+./setup.sh
+```
+
+To build and run the application for x86_64, use:
+
+```console
+./setup.sh
+make distclean
+wget -O /tmp/defconfig \
+     https://raw.githubusercontent.com/unikraft/app-lua/stable/defconfigs/qemu-x86_64-initrd
```

Comment:
Use the same link as below: https://raw.githubusercontent.com/unikraft/catalog-core/refs/heads/scripts/lua/scripts/defconfig/qemu.x86_64